src/weights.rs

```diff
@@ -64,7 +64,7 @@ impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 	fn initialize_reward_vec(x: u32) -> Weight {
 		(50_432_000 as Weight)
 			// Standard Error: 21_000
-			.saturating_add((72_298_000 as Weight).saturating_mul(x as Weight))
+			.saturating_add((143_109_000 as Weight).saturating_mul(x as Weight))
```

Comment:
This is a considerable jump, is this expected?

Reply:
Yes, its a mistake, I will do anotehr PR fixing it